src/ray/raylet/worker_pool.cc

```diff
@@ -295,6 +295,12 @@ Process WorkerPool::StartWorkerProcess(
                                   worker_command_args.begin() + 3);
       }
     }
+
+    WorkerCacheKey env = {dynamic_options, override_environment_variables,
+                          serialized_runtime_env};
+    int runtime_env_hash = env.IntHash();
+    std::string runtime_env_hash_str = std::to_string(runtime_env_hash);
```

Comment:
```suggestion
    const std::string runtime_env_hash_str = std::to_string(env.IntHash());
```

Actually ignore the above message and do,